File: 프로그래머스/unrated/181878. 원하는 문자열 찾기/원하는 문자열 찾기.java
class Solution {
    public int solution(String myString, String pat) {
        int answer = 0;
        
        myString = myString.toLowerCase();
        pat = pat.toLowerCase();
        System.out.print(myString);
        
        if (myString.indexOf(pat) >= 0) {
            answer = 1;
        }
        
        return answer;
    }
}

File: JS_Algo/프로그래머스/unrated/181878. 원하는 문자열 찾기/원하는 문자열 찾기.js
function solution(myString, pat) {
    let answer = 0;
    
    let a = myString.toLowerCase()
    let b = pat.toLowerCase()

    let arr = a.split(b)
    
    console.log(arr)
    
    if (arr.length > 1) {
        answer = 1;
    }
    
    return answer
}